src/mapbox/extended_geolocate_control.js

```diff
@@ -1,7 +1,10 @@
 import * as Geolocation from '../libs/geolocation';
 import Telemetry from '../libs/telemetry';
+import { openPendingGeolocateModal } from 'src/modals/GeolocationModal';
+import { get } from 'src/adapters/store';
 
 import { GeolocateControl } from 'mapbox-gl--ENV';
+import { set } from '../adapters/store';
```

Comment:
You can merge this line with the one above. Also, it could be useful to give these two functions some more explicit names locally.

Reply:
thanks